packages/grafana-o11y-ds-frontend/src/TemporaryAlert.tsx

```diff
@@ -0,0 +1,61 @@
+import { css } from '@emotion/css';
+import React, { useEffect, useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, useTheme2 } from '@grafana/ui';
+
+export enum AlertSeverity {
```

Comment:
I would use `AlertSeverity` from @grafana/ui instead of defining new types https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Alert/Alert.tsx#L14

Reply:
Definitely, I like that too. The reason behind the suggestion is that you can see this as "extending" the underlying Alert component with a more specific component, analogous to OOP. By using the same type from Alert you would be respecting and propagating the conventions from ui/Alert.
If another developer introduces another specific Alert component, they would either introduce a new type, use this one from TemporaryAlert or use the same as Alert. If everyone's use the same type as Alert is easier to be consistent.
Ideally this should be done in Alert.
In any case, also not a hard opinion, mainly a good practice suggestion.